### PR TITLE
[Imaging Browser] Added tooltips to ImagePanel file names

### PR DESCRIPTION
**Pull request by davidblader**
https://redmine.cbrain.mcgill.ca/issues/12786
As the redmine ticket mentions this feature was recommended in the comments of #2776 
To test: make sure file name appears in tooltip in the manner shown below
image

### CODE REVIEW

**Comment by driusan** [comment]
I'm just requesting a review from @rathisekaran because GitHub told me to do it..

**Comment by driusan** [comment]
@davidblader Is this still relevent after you've changed to a single column?

**Comment by davidblader** [comment]
> @davidblader Is this still relevent after you've changed to a single column?

i believe so since i don't think we are planning on changing the size of the image box itself, and even if we did it is not guaranteed that the filename will fit entirely within it.

**Comment by driusan** [comment]
@davidblader but is it going to conflict and going to have to be redone if you've basically redone the whole GUI?

Also, I just noticed this is going to the 17.1-dev branch, which is definitely not right..

**Comment by davidblader** [comment]
shouldn't cause any conflicts
i'll rebase according to whatever appropriate branch

File: modules/imaging_browser/jsx/ImagePanel.js [hunk 1]
```diff
@@ -42,12 +42,15 @@ var ImagePanelHeader = React.createClass({
                 </div>
             </div>
             );
+    $(".panel-title").tooltip();
```
Comment (HenriRabalais) [inline modules/imaging_browser/jsx/ImagePanel.js]: It may be best to declare this in 'componentDidMount()'

**Comment by davidblader** [comment]
@cmadjar should be fixed now

**Comment by ridz1208** [comment]
@cmadjar please re-review

**Comment by cmadjar** [comment]
> I'm just a little confused as the reasoning for this pull-request. If I'm not mistaken it seems that the only purpose of the tooltip is to re-iterate the Filename that is already displayed on the page with no additional information. Perhaps I'm overlooking something, but isn't that a bit redundant?

@HenriRabalais To answer your question, it is possible that the name of the file becomes very long and then not entirely displayed at the top of the pic (specially true for processed datasets that takes the name of the input file and add more text to the new file name) Example:
"preventad_123456_NAPBL00_dti_004_DTIPrepDTIColorFA_001.mnc" So I guess this is when the tooltip becomes handy.

@davidblader is there a plan to truncate then the filename displayed at the top so that the header info button does not look off like in the screenshot attached?
screen shot 2018-03-19 at 12 13 29 pm

**Pull request merged**